lib/ansible/modules/cloud/openstack/os_group.py

```diff
@@ -42,7 +42,7 @@
      description:
         - Domain id to create the group in if the cloud supports domains.
      required: false
-     default: None
+     default: default
```

Comment:
If we have the 'if domain_id' on line 149 then this can continue to default to None